homework_03/14_martin_damyanov/tests.py

```diff
@@ -0,0 +1,81 @@
+from homework import *
+import pytest
+
+def test_create_point():
+    p1 = Point(3.1, 3.5)
+    assert isinstance(p1, Point)
+
+def test_create_invalid_point():
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        p2 = Point('Gosho', 5.7)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_create_circle():
+    p = Point(5.4, 4.5)
+    c = Circle(p, 5.0)
+    assert isinstance(c, Circle)
+
+def test_create_invalid_circle():
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        c1 = Circle(5, 6)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        p = Point(2.5, 9.7)
+        c2 = Circle(p, 'radius')
+
+    assert pytest_wrapped_e.type == SystemExit
```

Comment:
Very cool that you have added tests for the creation of valid and invalid instances of the two classes. Good job!